pyproject.toml

```
@@ -0,0 +1,122 @@
+# For more information about configuring project metadata for the
+# setuptools build backend, please see
+# https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools"]
+
+[project]
+authors = [
+    { name = "Cybersecurity and Infrastructure Security Agency", email = "github@cisa.dhs.gov" }
+]
+# See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers = [
+    # How mature is this project? Common values are
+    #   3 - Alpha
+    #   4 - Beta
+    #   5 - Production/Stable
+    "Development Status :: 3 - Alpha",
+    "Environment :: Console",
+    # Indicate who your project is intended for
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    # Specify the Python versions you support here. In particular, ensure
+    # that you indicate whether you support Python 2, Python 3 or both.
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+dependencies = [
+    "Babel",
+    "PyGithub",
+    "python-dateutil",
+    "pytimeparse",
+]
+description = "GitHub Action to rebuild repositories that haven't been built in a while."
+dynamic = ["readme", "version"]
+keywords = ["apb"]
+license = "CC0-1.0"
+name = "apb"
+requires-python = ">=3.10"
+
+[project.optional-dependencies]
+# IMPORTANT: Keep type hinting-related dependencies of the dev section
+# in sync with the mypy pre-commit hook configuration (see
+# .pre-commit-config.yaml). Any changes to type hinting-related
+# dependencies here should be reflected in the additional_dependencies
+# field of the mypy pre-commit hook to avoid discrepancies in type
+# checking between environments.
+dev = [
+    "types-python-dateutil",
+    "types-requests",
+]
+test = [
+    "coverage",
+    "coveralls",
+    "pre-commit",
+    "pytest-cov",
+    "pytest",
+]
+
+[project.scripts]
+apb = "apb.entrypoint:main"
+[project.urls]
```

Add a blank line between sections for readability:
```suggestion
apb = "apb.entrypoint:main"

[project.urls]
```